From 61ba722bb8003902e043ddec589ac7fee042ba5b Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Thu, 8 Dec 2022 13:15:18 +0100
Subject: [PATCH] Disable gtk-doc checks
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

gtkdoc-check on el8 only supports make, not meson:

  8/8 libmbim-glib-doc-check FAIL            0.04s   exit status 1
  >>> DOC_MODULE=libmbim-glib MALLOC_PERTURB_=253 DOC_MAIN_SGML_FILE=libmbim-glib-docs.xml /usr/bin/gtkdoc-check
  ――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――
  Cannot find DOC_MODULE in Makefile
  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
---
 docs/reference/libmbim-glib/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/reference/libmbim-glib/meson.build b/docs/reference/libmbim-glib/meson.build
index 335d860..c203138 100644
--- a/docs/reference/libmbim-glib/meson.build
+++ b/docs/reference/libmbim-glib/meson.build
@@ -69,5 +69,5 @@ gnome.gtkdoc(
   fixxref_args: fixxref_args,
   content_files: [sections_txt, version_xml],
   install: true,
-  check: true,
+  check: false,
 )
-- 
2.38.1

